bindings/rust/src/lib.rs

```diff
@@ -1107,25 +1020,22 @@ macro_rules! sig_variant_impl {
 
                 // TODO - check msg uniqueness?
 
-                let pool = mt::da_pool();
-                let (tx, rx) = channel();
-                let counter = Arc::new(AtomicUsize::new(0));
-                let valid = Arc::new(AtomicBool::new(true));
-
-                let n_workers = core::cmp::min(pool.max_count(), n_elems);
-                for _ in 0..n_workers {
-                    let tx = tx.clone();
-                    let counter = counter.clone();
-                    let valid = valid.clone();
+                let counter = AtomicUsize::new(0);
+                let valid = AtomicBool::new(true);
+                let acc = Mutex::new(None::<Pairing>);
+                let mut gtsig = blst_fp12::default();
```

Comment:
`blst_fp12` has no `Default`.

The issue is that on Fp12 additive group, the default should be 0.
But on GT multiplicative subgroup, the one we're interested for pairing, the default should be 1.

Reply:
> blst_fp12 has no Default.

It obviously does or code wouldn't compile. Also this is exactly the same code as was present before, see old line number 1156.